.gitignore

```diff
@@ -1,5 +1,8 @@
 node_modules
+dist
 .cache
+.env*
+.next
```

Comment:
Small note: I moved these to the top-level gitignore, and then realized that prettier doesn't look for gitignore files recursively (https://github.com/prettier/prettier/issues/4081), so I had to mention `.next` and `.dist` in all separate `.prettierignore`.

Still a win, but much smaller than I hoped. Maybe `lint:prettier` and `format` jobs should be whitelisting files, `prettier --check src test`.